Phonebook.Frontend/src/app/shared/directives/feedback-drawer/feedback-drawer-sheet/feedback-drawer-sheet.component.html

```diff
@@ -14,7 +32,7 @@ <h3 mat-line i18n="FeedbackDrawerSheetComponent|Title of the Feedback Drawer@@Fe
     >
       <span
         i18n="FeedbackDrawerSheetComponent|Button for contacting the private Organization@@FeedbackDrawerSheetComponentButtonPrivateOrganization"
-        >Contact your private Organization</span
+        >Contact your private organization repository</span
```

Comment:
```suggestion
        >Contact your private organization</span
```
why should somebody speak to code? ^^ :D

Reply:
Thats true, I will change that